FSJ_django20_project/FSJ/templates/FSJ/coord_adjudicator_list.html

```diff
@@ -0,0 +1,40 @@
+{% extends "FSJ/base_with_banner.html" %}
+{% load i18n %}
+
+{% block heading %}
+    {# Translators: Adjudicator page header #}
+    {% trans "Adjudicators" %}
+{% endblock %}
+
+
+{% block content %}
```

Comment:
template looks very similar to the student list. I would like to see them generalized and put together as something extending the base or base with banners, so that we can use a general template and include the appropriate list of objects...however, that may be best left until we have some solid filterable / sortable table from Jonah.

Reply:
Haven't found a good way to iterate over fields in a table yet, so some information for students would be left out if I used the same template for both. Can change if that's preferable.

Reply:
Might be best to wait to see what Jonah delivers before we go down the hole of assessing what we should do to simplify this. Mostly wanted to verbalize it and keep it in the back of our minds for now.